### DISCUSSION TITLE
Docs for Vite configuration?

### DISCUSSION THREAD

**Discussion by lindsaykwardell** [nuxt/nuxt#18151] [General]
I recently spun up a Nuxt 3 app, and wanted to configure it to use Vite with the intent of using a Vite plugin (in this case, [`vite-plugin-elm`](https://www.npmjs.com/package/vite-plugin-elm)). However, going through the documentation for Nuxt 3, I couldn't find instructions on how to configure Vite. I was eventually able to figure it out exploring the available types in `nuxt.config.ts`, but I was confused that there wasn't a page for this on the Nuxt site. Is this something that's in progress?

**Comment by diegolepore** [comment]
Hi @lindsaykwardell 👋 

I was running into a similar issue, and I just found out something that worked for me in the docs (not very clear, though): https://v3.nuxtjs.org/getting-started/bridge/#feature-flags

Basically you just have to add a vite object inside the nuxt config object:

```
import { defineNuxtConfig } from 'nuxt3'

export default defineNuxtConfig({
  vite: {
    // Config for Vite
  }
})
```

I hope it helps 😃 👍 
Cheers!
